### DISCUSSION TITLE
Get last row elements in Grid

### DISCUSSION THREAD

**Discussion by Marius-brt** [mantinedev/mantine#3231] [Q&A]
How can I get dynamicaly the last row elements in Grid ?
I didn't find any answer to do this in css.

**Comment by rtivital** [comment]
What do you mean by "get dynamicaly the last row elements in Grid"?

**Reply by Marius-brt** [reply]
By dynamicaly I mean that auto update when the size of the window change. I want to modify the css of selected elements on the pictures (its the same grid just the size of the window change between the two photos)

![Capture](https://user-images.githubusercontent.com/31184204/209367506-4993e94c-8b31-41bb-9b92-c6930fbb58df.PNG)
![Capture2](https://user-images.githubusercontent.com/31184204/209367516-51d41a20-0655-47a0-a790-0bbe546c6f97.PNG)

**Reply by rtivital** [reply]
And what exactly should happen?
